psse/psse-converter/src/main/java/com/powsybl/psse/converter/SwitchedShuntCompensatorConverter.java

```diff
@@ -0,0 +1,266 @@
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.ShuntCompensator;
+import com.powsybl.iidm.network.ShuntCompensatorAdder;
+import com.powsybl.iidm.network.ShuntCompensatorNonLinearModelAdder;
+import com.powsybl.iidm.network.Terminal;
+import com.powsybl.iidm.network.VoltageLevel;
+import com.powsybl.iidm.network.util.ContainersMapping;
+import com.powsybl.psse.model.PsseException;
+import com.powsybl.psse.model.PsseVersion;
+import com.powsybl.psse.model.pf.PsseSwitchedShunt;
+import static com.powsybl.psse.model.PsseVersion.Major.V35;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class SwitchedShuntCompensatorConverter extends AbstractConverter {
+
+    public SwitchedShuntCompensatorConverter(PsseSwitchedShunt psseSwitchedShunt, ContainersMapping containerMapping, Network network, PsseVersion version) {
+        super(containerMapping, network);
+        this.psseSwitchedShunt = psseSwitchedShunt;
+        this.version = version;
+    }
+
+    public void create() {
+        List<ShuntBlock> shuntBlocks = defineShuntBlocks(psseSwitchedShunt, version);
+        if (shuntBlocks.isEmpty()) {
+            return;
+        }
+
+        String busId = getBusId(psseSwitchedShunt.getI());
+        String id = defineShuntId(psseSwitchedShunt, version);
+        VoltageLevel voltageLevel = getNetwork().getVoltageLevel(getContainersMapping().getVoltageLevelId(psseSwitchedShunt.getI()));
+
+        ShuntCompensatorAdder adder = voltageLevel.newShuntCompensator()
+            .setId(getShuntId(busId, id))
+            .setConnectableBus(busId)
+            .setSectionCount(defineSectionCount(psseSwitchedShunt.getBinit(), shuntBlocks));
+        ShuntCompensatorNonLinearModelAdder modelAdder = adder.newNonLinearModel();
+        shuntBlocks.forEach(shuntBlock -> {
+            for (int i = 0; i < shuntBlock.getN(); i++) {
+                modelAdder.beginSection()
+                    .setG(0.0)
+                    .setB(powerToShuntAdmittance(shuntBlock.getB(), voltageLevel.getNominalV()))
+                    .endSection();
+            }
+        });
+        modelAdder.add();
+        ShuntCompensator shunt = adder.add();
+
+        if (psseSwitchedShunt.getStat() == 1) {
+            shunt.getTerminal().connect();
+        }
+    }
+
+    public void addControl() {
+        String busId = getBusId(psseSwitchedShunt.getI());
+        String id = defineShuntId(psseSwitchedShunt, version);
+        ShuntCompensator shunt = getNetwork().getShuntCompensator(getShuntId(busId, id));
+
+        // Add control only if shunt has been created
+        if (shunt == null) {
+            return;
+        }
+
+        double vnom = shunt.getTerminal().getVoltageLevel().getNominalV();
```

Comment:
Same question everywhere: PSSE format does not support remote control?

Reply:
Yes, you are right. I fixed and i added a unitary test. Thanks.